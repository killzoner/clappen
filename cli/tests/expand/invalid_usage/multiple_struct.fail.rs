
fn main(){
    #[clappen::clappen(export = prefix)]
    mod m1 {
        pub struct A {}
        pub struct B {}
    }
}
