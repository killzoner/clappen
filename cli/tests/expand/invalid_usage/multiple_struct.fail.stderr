error: clappen must have a unique struct definition
 --> tests/expand/invalid_usage/multiple_struct.fail.rs:3:24
  |
3 |     #[clappen::clappen(export = prefix)]
  |                        ^^^^^^^^^^^^^^^
